editor/scene/texture/texture_editor_plugin.cpp

```diff
@@ -286,6 +400,35 @@ TexturePreview::TexturePreview(Ref<Texture2D> p_texture, bool p_show_metadata) {
 		add_child(channel_selector);
 	}
 
+	// Add right upper buttons
+	right_upper_corner_container = memnew(HBoxContainer);
+	right_upper_corner_container->set_h_size_flags(Control::SIZE_SHRINK_END);
+	right_upper_corner_container->set_v_size_flags(Control::SIZE_SHRINK_BEGIN);
+	add_child(right_upper_corner_container);
+	zoom_out_button = memnew(Button);
+	zoom_out_button->connect("pressed", callable_mp(this, &TexturePreview::on_zoom_out_pressed));
```

Comment:
```suggestion
	zoom_out_button->connect(SceneStringName(pressed), callable_mp(this, &TexturePreview::on_zoom_out_pressed));
```
For all uses of this signal